### PR TITLE
Login page design + email validation + scrolling on character details page

### PR DESCRIPTION
**Pull request by MicLamb**
Design of the login page
Regex on the email
Scrolling added on the character page

#29

### CODE REVIEW

**Comment by DrPandemic** [comment]
Should add a config to decide if the login button is enable or not by default

File: frontend/MCOO_gn_frontend/app/src/main/java/mcoo/mcoo_gn_frontend/CharacterActivity.java [hunk 1]
```diff
@@ -106,22 +122,34 @@ public void onClickEditCharacter(View v){
...
 
     public void onClickUpdateCharacter(View v){
-        textViewCharacterName.setEnabled(false);
-        textViewCharacterNationality.setEnabled(false);
-        textViewCharacterRace.setEnabled(false);
-        textViewCharacterProfession.setEnabled(false);
-        textViewCharacterClass.setEnabled(false);
-        textViewCharacterBelief.setEnabled(false);
-        textViewCharacterMaxHP.setEnabled(false);
-        textViewCharacterXP.setEnabled(false);
```
Comment (DrPandemic) [inline frontend/MCOO_gn_frontend/app/src/main/java/mcoo/mcoo_gn_frontend/CharacterActivity.java]: Nice refactore!

**Comment by Baltazar510** [comment]
Everything is awesome.

File: frontend/MCOO_gn_frontend/app/src/main/java/mcoo/mcoo_gn_frontend/MainActivity.java [hunk 2]
```diff
@@ -13,6 +19,26 @@
...
+                    else
+                        btnLogin.setEnabled(true);
+                }
+            }
+        });
+    }
+
+    private boolean isValidEmail(String email) {
+        Pattern pattern = Patterns.EMAIL_ADDRESS;
+        return pattern.matcher(email).matches();
```
Comment (DrPandemic) [inline frontend/MCOO_gn_frontend/app/src/main/java/mcoo/mcoo_gn_frontend/MainActivity.java]: Style : You could one line this check
```java
return Patterns.EMAIL_ADDRESS.matcher(email).matches();
```

**Comment by DrPandemic** [comment]
:+1:

**Comment by pellerinse2302** [comment]
:+1:

**Comment by Erikad94** [comment]
ITS ALIIIIIIIVE

**Pull request merged**